test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -160,7 +172,6 @@ bootstrap_remote_freebsd()
             libyaml
             ${pyyaml_pkg}
             ${jinja2_pkg}
-            ${cryptography_pkg}
```

Comment:
Why was this removed?

Reply:
not needed for 3.9 as it is part of the installed python, but moved as 3.11 does not have it, even as a package

Reply:
It's not a dependency of the `python39` package, but it is already installed.

Reply:
We'll need to keep this, until we drop FreeBSD 13.2, as noted above.